src/main/java/com/google/sps/model/Users.java

```diff
@@ -0,0 +1,31 @@
+package com.google.sps.model;
+
+import java.util.List;
+
+public class Users {
+    class UserTrip {
+        String title;
+        String linkToTrip;
```

Comment:
I would call this tripId

Reply:
I added Trip objects to a list in the User class and am thinking that we could get the trips ID when we click on the link.